From 718ddaea515d894abfccdb034b47f5fe9e20513f Mon Sep 17 00:00:00 2001
From: Parham Alvani <parham.alvani@gmail.com>
Date: Wed, 3 Dec 2014 10:18:46 +0330
Subject: [PATCH] Makefile changed due to the Eudyptula task no.03

Signed-off-by: Parham Alvani <parham.alvani@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ce70361..13519b1 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
 PATCHLEVEL = 18
 SUBLEVEL = 0
-EXTRAVERSION = -rc7
+EXTRAVERSION = -rc7-eudyptula
 NAME = Diseased Newt
 
 # *DOCUMENTATION*
-- 
1.9.1

